src/context/AppContext.js

```diff
@@ -0,0 +1,47 @@
+import React, {useContext, useState} from 'react'
+
+const Context = React.createContext();
+
+export const useApp = () => {
+  return useContext(Context)
+}
+
+const AppProvider = ({children}) => {
+  // global useState
+  const [events, setEvents] = useState([])
+  const [contacts, setContacts] = useState([])
+  const [error, setError] = useState('')
```

Comment:
I really like being able to share this error and loading data with all of the components